### DISCUSSION TITLE
eBUS network goes down when connecting eBUS Adapter Shield v5

### DISCUSSION THREAD

**Discussion by MatthiasValvekens** [john30/ebusd#1309] [Q&A]
Hello,

I have a heat pump installation at home consisting of a Vailliant aroTHERM VWL 35/5 AS, uniTOWER VWL 58 ... 128/5 IS and a VRC 700/6 control unit.

I got an eBUS Adapter Shield v5 to monitor/control the installation remotely. To figure out where to install it, I followed the eBUS wire from the uniTOWER's eBUS terminal to a point where it was plugged into a pair of 3-way Wago connectors (wired to the other devices in the eBUS network in parallel). I replaced those two Wagos with bigger ones and wired up my eBUS adapter similarly in parallel with the others; the cable run is less than 2m. For my tests, I connected the eBUS adapter to my network wirelessly, and powered it over USB. It's flashed with the latest firmware.

This is what I observe when the adapter is connected to the eBUS circuit:

 - I can connect to the adapter with ebusd, but it doesn't get any messages. The LEDs on the board also don't really react (only blue led blinks).
 - The adapter reports "no signal" as well
 - The connection between the uniTOWER and the VRC 700 control unit breaks down (control unit screen blanks, uniTOWER displays a communication error) -- a bit like #967, except that in my case it happens consistently when the adapter is connected
 - This is also what I see when I short out the two wires connected to the eBUS circuit, by the way (not surprising given that all the devices are connected in parallel)
 - Whether the eBUS adapter is connected to power or not does not seem to matter, the effects on the rest of the eBUS network is the same.

For completeness, here's the output of `ebusctl info`:

```
version: ebusd 23.3.23.3
update check: OK
device: 10.6.0.6:9999, TCP, enhanced, firmware 1.1[460f].1[460f]
signal: no signal
reconnects: 0
masters: 1
messages: 11
conditional: 0
poll: 0
update: 4
address 31: master #8, ebusd
address 36: slave #8, ebusd
```

It's identical, whether the adapter is actually connected to the eBUS network or not.

As soon as I physically disconnect the eBUS terminal on the adapter, the other devices on the eBUS network immediately recover. What am I missing? Is it possible that I simply have a faulty unit, e.g. could there be a short in the eBUS terminal on the adapter, for example?

Happy to provide further details if needed.

Thanks!

**Comment by john30** [comment]
sounds pretty much like you got the wrong plug on the board as eBUS is never coming with a 3-way connector but always only 2. it also needs to say "eBUS" in the heater manual. everything else is simply wrong and using that could also damage your devices

**Reply by MatthiasValvekens** [reply]
Thanks for the reply!

So, perhaps I explained myself wrongly. The eBUS plug on the board takes two wires and is market "BUS"; the terminals are marked + and - (see first picture). I also cross-referenced with the schematics in the installer's manual to ensure that I got the correct plug, so I don't think that's it.

The + wire travels to a 3-way Wago, and the - wire travels to another 3-way Wago. From those Wagos there are similar wires travelling to the the other eBUS devices on the network (so they're connected in parallel); see 2nd picture (which is crappy due to the difficult angle). I replaced those two 3-ways with a 5-ways and plugged in my adapter in the same way as the other wires plugged into those connectors.

Does that make more sense, or am I still missing something obvious? :)

![IMG_1284](https://github.com/user-attachments/assets/9bdc102c-feec-415d-a53e-7dd5fb8e6951)
![IMG_1279](https://github.com/user-attachments/assets/c59ba4f2-387d-48f4-ac52-2dcb3024a05d)

EDIT: picture of schematic from the manual, for completeness:

![IMG_1288](https://github.com/user-attachments/assets/9672cea2-4c5f-4b04-ab33-397f7e38f0d3)

**Comment by john30** [comment]
oh dear, don't shortcut anything by intention - this might ruin your power source or do other damage.
anyway, you might be facing the seldom case of a device being dead on arrival. it is very rare as every adapter is tested on the eBUS side including communication before delivery (and I have proof of that). so if you're sure that you did't ruin the adapter already yourself (e.g. by creating a shortcut on the adapter itself somehow), ask for replacement (and best add me in cc)

EDIT: are you sure that you got the right cable as your picture shows several ones with identical green cable? you might just want to connect the adapter directly to the eBUS plug on the circuit just to be sure

**Reply by MatthiasValvekens** [reply]
Thanks again for your comments. I'm sure the cable is the right one; it's hard to tell from the pictures, but the Wago connectors are located in view of the eBus terminal on the board, so checking that was simple enough :). 

Regarding replacing the adapter: even if the adapter is at fault, then all things considered chances are that I fried it doing something dumb. I'll eat the cost of replacing it (but thank you for offering). But that'll be something to deal with a couple weeks from now when I actually have time to work on this again.

Thanks once more!

**Comment by OmidTahouri** [comment]
@MatthiasValvekens curious to know where you got to with this. Did you attempt anything else?

I had the [small C6 adapter](https://adapter.ebusd.eu/v5-c6/index.en.html) adapter connected to my "Vaillant aroTHERM plus 7" system's "VR 940f" main control unit _thing_, working great for about 5 weeks. Out of nowhere, I suddenly noticed my thermostat on the wall was completely off, so I checked the main unit to see "Communication fault" on the screen and the system quite obviously unavailable/down.

Power cycling achieved nothing. The control unit would sometimes look like it was booting up, but would end up tripping the breaker after <10 seconds. Other times, the boot up would result in the same "Communication fault" error message. I tried many basic troubleshooting steps, including powering all components off, including the ASHP outside. I observed the same behaviour even with the eBUS adapter disconnected from USB power but still connected to the BUS terminal. My only fix was to fully disconnect the eBUS adapter, at which point the system powered on without any issue.

It's not clear to me why this happened randomly, without any changes and after multiple weeks of a functioning adapter.

I acknowledge that the adapter stopped reporting data some time before this incident occurred, but I believe that's due to the absence of some license/key (?). The adapter was alive and healthy from an MQTT perspective, just not reporting anything useful (which is how I learned of a potential key.

I'm now hesitant to reconnect the adapter until I learn more. In case it matters, I appear to have two devices connected to the BUS, an outdoor temperature probe/sensor and, presumably, the main thermostat (?). The eBUS adapter was the third device that I introduced with what I thought was a neat installation using a couple of wago connectors. Did something overload the BUS terminal? I'd like to think/hope this isn't all caused by the absence of the license/key.

**Reply by OmidTahouri** [reply]
Following up with a lengthy update, but I at least come with data 😅  The adapter is (was) integrated over MQTT into Home Assistant, so its own status topics were being recorded throughout. With my Home Assistant instance fronted by MCP, I was able to dig in a bit. Apologies for the clearly AI-generated response, but I promise it's interesting:

**It wasn't sitting quietly idle before the failure — it was restarting every few minutes.**

`device_running` logged **206 transitions** between 31 Aug and 3 Sep:

| | |
|---|---|
| `on` duration | median **241 s** (min 45 s) |
| `off` duration | median **15.3 s** |
| transitions per day | 67, 15, 89, 34 |

A consistent 15-second gap is a restart cycle, not a network or broker dropout.

That makes me doubt my own licence/key theory. A missing config key should leave a healthy, stable adapter that simply can't decode anything useful. What I actually had was a device power-cycling itself every four minutes for three days.

**The recording of the incident itself matches what I saw at the unit:**

- **12:17** — heating system goes unavailable
- **13:41:23** — adapter's last MQTT message ever, i.e. USB power removed
- system stays faulted for a further **~22 minutes**
- **14:03:40** — system recovers, immediately after the adapter came off the BUS terminal

So there's a clear window where the adapter was unpowered and completely silent and the controller was *still* faulted. It only came back when the thing was physically off the pair.

**On my own "did I overload the BUS terminal?" question — I now doubt it.** The C6 is class 0 under the eBUS spec, fully galvanically isolated, and USB-powered rather than bus-powered. It also **ran as the third device for five weeks without incident**, and a static current-budget limit should bite on day one.

What fits better is a degrading fault in the adapter's bus-side front end or its isolation barrier — something that became a load, or a partial short, across the pair regardless of whether it was powered. That would also account for the breaker trips, which feel electrical rather than protocol-level.

**Questions**

1. Has anyone seen a C6 *degrade in service* rather than fail dead on arrival? Mine worked normally for five weeks first, which is what makes me doubt the DOA explanation earlier in this thread.
2. By design, is the C6 electrically inert on the bus once USB power is removed?
3. Can the config/licence key affect anything bus-side, or is it purely decode-side?

@MatthiasValvekens — did you end up replacing yours, and did a new one behave?

I'm not reconnecting mine to find out; that test costs me heating and hot water.
